src/web/components/Figure.tsx

```diff
@@ -129,7 +122,8 @@ const roleCss = {
 	`,
 };
 
-const decidePosition = (role: RoleType | 'richLink') => {
+// Used for vast majority of layouts.
+export const standardRoleStyler = (role: RoleType | 'richLink') => {
```

Comment:
granted `decidePosition` wasn't a great name for the prop (it does a lot more than just decide position) but I am not sure about `standardRoleStyler`. How about something like `roleBasedStyles` ?

Reply:
👍 perhaps `defaultRoleBasedStyles`?